strapi/strapi
# Translate en.json into traditional Chinese.

### What does it do?

Translate en.json into traditional Chinese(zh.json).
Enable the traditional Chinese interface language. 

### Why is it needed?

Solve the problem of no Chinese interface language.

### How to test it?

Switch the interface language to Chinese and check the display status.
No additional features are added, and no new test code is required.

### Related issue(s)/PR(s)

none
